src/main/java/com/service/OrderStatusService.java

```diff
@@ -0,0 +1,43 @@
+package com.service;
+
+import com.dao.OrderDAO;
+import com.model.Order;
+
+public class OrderStatusService {
+
+    private OrderDAO orderDAO = OrderDAO.getInstance();
+    private static OrderStatusService instance;
+
+    private OrderStatusService() {
+    }
+
+    public void setOrderDAO(OrderDAO orderDAO) {
+        this.orderDAO = orderDAO;
+    }
+
+    public static OrderStatusService getInstance() {
+        if (instance == null) {
+            instance = new OrderStatusService();
+        }
+        return instance;
+    }
+
+
+    public void confirmOrder(int id) {
+        Order order = orderDAO.getById(id).get();
```

Comment:
If optional doesn't contain element .get() could throw NoSuchElementException. It would be better to check element existence with .ifPresent() first and then throw custom exception with description of error